pkg/collector/corechecks/oracle-dbm/sql_wrappers.go

```diff
@@ -52,12 +52,20 @@ func handlePrivilegeError(c *Check, err error) (bool, error) {
 		return isPrivilegeError, err
 	}
 
-	links := map[hostingCode]string{
-		selfManaged: "https://docs.datadoghq.com/database_monitoring/setup_oracle/selfhosted/#grant-permissions",
-		rds:         "https://docs.datadoghq.com/database_monitoring/setup_oracle/rds/#grant-permissions",
-		oci:         "https://docs.datadoghq.com/database_monitoring/setup_oracle/autonomous_database/#grant-permissions",
+	var link string
+	switch c.hostingType {
+	case selfManaged:
+		if c.multitenant {
+			link = "https://docs.datadoghq.com/database_monitoring/setup_oracle/selfhosted/?tab=multitenant#grant-permissions"
```

Comment:
just a nit, but this `https://docs.datadoghq.com/database_monitoring/setup_oracle/` prefix should probably be a constant